src/ValidatableExtensions/ValidatableExtensions.Strings.cs

```diff
@@ -200,4 +200,34 @@ public static partial class ValidatableExtensions
 
         return ref validatable;
     }
+
+    /// <summary>
+    /// Throws an exception if the string contains the given <paramref name="otherString"/>
+    /// Default <paramref name="comparisonType"/> is Ordinal.
```

Comment:
Perhaps something like this so the values here are navigatable? (H&B)
```suggestion
    /// The default <see cref="StringComparison"/> type is <see cref="StringComparison.Ordinal"/>.
```